website/pages/en/index.js

```diff
@@ -138,17 +141,20 @@ class Index extends React.Component {
                 <div className="blockImage">
                   <div>
                     <MarkdownBlock>
-                      {typeAnnotationExample}
+                      {asyncExample}
                     </MarkdownBlock>
                   </div>
                 </div>
                 <div className="blockContent">
                   <h2>
-                    Type Annotations
+                    Async
                   </h2>
                   <div>
                     <MarkdownBlock>
-                      Type annotations allow for code to be explicitly typed on parameters, class member variables and return values.
+                      **Async is not multithreading**. HHVM executes a program's code in one main request thread.
```

Comment:
This is important, but here it's probably to swap these around to describe what it is, before what it isn't